src/middleware/err_handlers.rs

```diff
@@ -37,27 +37,21 @@ type ErrorHandler<B> = dyn Fn(ServiceResponse<B>) -> Result<ErrorHandlerResponse
 ///
 /// # Examples
 /// ```
-/// use actix_web::middleware::{ErrorHandlers, ErrorHandlerResponse};
-/// use actix_web::{web, dev, App, HttpRequest, HttpResponse, Result};
-/// use actix_web::http::{StatusCode, header};
-///
-/// fn render_500<B>(mut res: dev::ServiceResponse<B>) -> Result<ErrorHandlerResponse<B>> {
-///     res.response_mut()
-///        .headers_mut()
-///        .insert(header::CONTENT_TYPE, header::HeaderValue::from_static("Error"));
+/// use actix_web::http::{header, StatusCode};
+/// use actix_web::middleware::{ErrorHandlerResponse, ErrorHandlers};
+/// use actix_web::{dev, web, App, HttpResponse, Result};
 ///
+/// fn add_error_header<B>(mut res: dev::ServiceResponse<B>) -> Result<ErrorHandlerResponse<B>> {
+///     res.response_mut().headers_mut().insert(
+///         header::CONTENT_TYPE,
+///         header::HeaderValue::from_static("Error"),
+///     );
 ///     Ok(ErrorHandlerResponse::Response(res.map_into_left_body()))
 /// }
 ///
 /// let app = App::new()
-///     .wrap(
-///         ErrorHandlers::new()
-///             .handler(StatusCode::INTERNAL_SERVER_ERROR, render_500),
-///     )
-///     .service(web::resource("/test")
-///         .route(web::get().to(|| HttpResponse::Ok()))
-///         .route(web::head().to(|| HttpResponse::MethodNotAllowed())
-///     ));
+///     .wrap(ErrorHandlers::new().handler(StatusCode::INTERNAL_SERVER_ERROR, add_error_header))
```

Comment:
This reformatting was also performed by `cargo fmt` by copy-pasting into a fresh repo. I'm not sure if we keep doc comments here tighter than 100 characters so I'll happily unformat if we prefer that!